src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -190,20 +190,9 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
                     return rank <= 5lu;
                 });
 
-        pass_config->set_callback<ngraph::pass::ConvertReduceSumToPooling>(
-            [](const_node_ptr &node) -> bool {
-                return disableReduceDecomposition<ngraph::opset1::ReduceSum>(node);
-            });
-
-        pass_config->set_callback<ngraph::pass::ConvertReduceMeanToPooling>(
-            [](const_node_ptr &node) -> bool {
-                return disableReduceDecomposition<ngraph::opset1::ReduceMean>(node);
-            });
-
-        pass_config->set_callback<ngraph::pass::ConvertReduceMaxToPooling>(
-            [](const_node_ptr &node) -> bool {
-                return disableReduceDecomposition<ngraph::opset1::ReduceMax>(node);
-            });
+        pass_config->disable<ngraph::pass::ConvertReduceSumToPooling>();
+        pass_config->disable<ngraph::pass::ConvertReduceMeanToPooling>();
+        pass_config->disable<ngraph::pass::ConvertReduceMaxToPooling>();
```

Comment:
I think we should disable it only when `support_immad == true`.

Reply:
Applied.